src/main/java/org/ohdsi/circe/check/checkers/Comparisons.java

```diff
@@ -101,19 +106,26 @@ public static Predicate<Concept> compare(Concept source) {
                 .append(concept.vocabularyId, source.vocabularyId)
                 .build();
     }
-
     public static int compareTo(ObservationFilter filter, Window window) {
-
-        int range1 = filter.postDays + filter.priorDays;
-        int range2Start = 0, range2End = 0;
-        if (Objects.nonNull(window.start) && Objects.nonNull(window.start.days)) {
-            range2Start = window.start.coeff * window.start.days;
-        }
-        if (Objects.nonNull(window.end) && Objects.nonNull(window.end.days)) {
-            range2End = window.end.coeff * window.end.days;
+        int range1 , range2Start , range2End;
+        if(Objects.nonNull(window.start) && Objects.nonNull(window.start.days)){
+                range1 = filter.postDays + filter.priorDays;
+                range2Start = window.start.coeff * window.start.days;
+                range2End = Objects.nonNull(window.end) && Objects.nonNull(window.end.days) ? window.end.coeff * window.end.days : 0;
+        }else {
+                range1 = (filter.postDays + filter.priorDays) * 24 * 60 * 60;
+                range2Start = getTimeInSeconds(window.start);
+                range2End = getTimeInSeconds(window.end);
```

Comment:
This is where the 4sp indent is glaring, so before you have to go back and change all of it back to 2sp, i'd adjust your formatting.